packages/components/stories/Button.stories.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import { Button, Icon } from '../src/components';
+import { Button, Icon, Lock } from '../src/components';
```

Comment:
IMO, we should not export **all** SVG components at the root level of UI-Toolkit. I am afraid that one day we will have a collision of component names. For example if one day we want to add a new **List** React UI component (to manage `<ui><li>` tags), then the name will collide with the SVG component `List`.

It would be nice to have something like :
```typescript
import { Icons } from '../src/components';
or
import { SvgIcons } from '../src/components';

 <Button iconLeft={ <Icons.Lock .../> } >
 or
 <Button iconLeft={ <SvgIcons.Lock .../> } >
```

Reply:
Yeah, you're right 👍 

I think exporting all of them under the `Icons` umbrella and accessing from there is good. Updating.